src/main/webapp/js/instructorFeedbackResultsAjaxByQuestion.js

```diff
@@ -50,6 +50,12 @@ $(document).ready(function() {
                 $(panelHeading).click(toggleSingleCollapse);
                 $(panelHeading).trigger('click');
 
+                // look for .no-response in panel body that is set by the backend
+                // switch panel from panel-info to panel-default (gray), if found.
```

Comment:
Better. Try to explain the intended outcome and rationale rather than how the code works. Sometimes we can rewrite the code to make the comment unnecessary. e.g.
```
if(isPanelSetAsEmptyByBackend(panelbody){
   displayAsEmptyPanel(panelbody)
}
```